File: .go/cli/ext/ext.go
/*
 * Copyright (c) 2025.
 * Created by Andy Pangaribuan (iam.pangaribuan@gmail.com)
 * https://github.com/apangaribuan
 *
 * This product is protected by copyright and distributed under
 * licenses restricting copying, distribution and decompilation.
 * All Rights Reserved.
 */

package ext

import (
	"squirrel/arg"
	"squirrel/util"

	"github.com/wissance/stringFormatter"
)

func CLI() {
	helpMessage := stringFormatter.FormatComplex(`
info : execute ext cli
usage: sq ext

{commands}
  rgo   robot go
`, map[string]any{
		"commands": util.ColorBoldGreen("commands:"),
	})

	arg.Watch("sq ext", helpMessage, helpMessage).
		Add("rgo", "", cliRgo).
		Exec()
}

File: src/cli/ext.rs
/*
 * Copyright (c) 2026.
 * Created by Andy Pangaribuan
 *
 * This product is protected by copyright and distributed under
 * licenses restricting copying, distribution and decompilation.
 * All Rights Reserved.
 */

pub fn run(args: &[String]) {
    if args.is_empty() || args[0] == "--help" || args[0] == "-h" {
        println!("info : execute ext cli");
        println!("usage: sq ext");
        return;
    }

    println!("ext command executed with args: {:?}", args);
}
